src/shims/unix/linux/eventfd.rs

```diff
@@ -72,6 +106,8 @@ impl FileDescription for Event {
                 Endian::Big => self.counter.to_be_bytes(),
             };
             self.counter = 0;
+            // When any of the event is happened, we check and update the status of all supported flags.
```

Comment:
I don't see any connection between the comment and the code...? Which "flags"?

Reply:
"flags" and "events" kind of being used interchangeably here. The flags/events are the ``EPOLLIN``, ``EPOLLOUT``, ``EPOLLRDHUP`` etc. I will change it to event instead.